src/com/palmergames/bukkit/towny/TownySettings.java

```diff
@@ -3485,5 +3485,9 @@ public static int getNewTownMinDistanceFromTownPlots() {
 	public static int getNewTownMinDistanceFromTownHomeblocks() {
 		return getInt(ConfigNodes.TOWN_NEW_TOWN_MIN_DISTANCE_FROM_TOWN_HOMEBLOCK);
 	}
+	
+	public static int getMinAdjacentBlocks() {
+		return getInt(ConfigNodes.TOWN_MIN_ADJACENT_BLOCKS);
```

Comment:
```suggestion
		return Math.min(3, getInt(ConfigNodes.TOWN_MIN_ADJACENT_BLOCKS));
```

Reply:
I'm not certain if this import is present yet. I don't think we should allow more than 3 to be set. This takes away their option.